templates/todo/web/react-fluent/entrypoint.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
The Dockerfile we use has `bash`, yeah?

Reply:
I will double check.  I was getting some errors that we're using bash-isms here.

Reply:
Ask chatgpt to convert it for you.